### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Mkarthik2002**
testing

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Connect GitHub with DevRev, and ensure your work stays in sync between the two systems, such that issues and statuses are always real-time and driven by code activity. GitHub for DevRev is a snap-in that has been used by several organizations, and ties DevRev issues closely with GitHub activity, eliminating the need for humans to manage issues.
 
-Why should you un-manage your work? You should use this snap-in if you want to reduce work about work. Let machines manage issues, including creating and updating statuses, so developers can focus on work that matters. You’ll resonate with this, if you have ever:
+Why should you un-manage your work? You should use this snap-in if you want to reduce work about work. Let machines manage issues, including creating and updating , so developers can focus on work that matters. You’ll resonate with this, if you have ever:
```
Comment (Mkarthik1234) [inline README.md:4]: this is wrong

File: README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
```
Comment (Mkarthik1234) [inline README.md:3]: this need to be modified

File: README.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 Connect GitHub with DevRev, and ensure your work stays in sync between the two systems, such that issues and statuses are always real-time and driven by code activity. GitHub for DevRev is a snap-in that has been used by several organizations, and ties DevRev issues closely with GitHub activity, eliminating the need for humans to manage issues.
```
Comment (Mkarthik1234) [inline README.md:1]: give one heading

File: README.md (hunk 1 above)
Comment (Mkarthik2002) [reply README.md:4]: yeah

File: README.md (hunk 2 above)
Comment (Mkarthik2002) [reply README.md:3]: okay i will do that

File: README.md (hunk 3 above)
Comment (Mkarthik2002) [reply README.md:1]: adding

**Pull request merged**